Makefile

```diff
@@ -19,7 +19,8 @@ install-generator: install ## Install dependencies for SDK code generator
 generate: install-generator  ## Generate the SDK from our public openapi spec
 	node_modules/.bin/openapi-generator-cli generate -i spec/public-api.yaml \
 		-g python \
-		-o ./generated
+		-o ./generated \
+		--additional-properties=packageName=groundlight_openapi_client
```

Comment:
This is the main change. Everything else is either auto-generated or CTRL-F changing references from `openapi_client` to `groundlight_openapi_client`